docs/configuration.md

```diff
@@ -1149,6 +1149,20 @@ Apart from these, the following properties are also available, and may be useful
   </td>
 </tr>
 <tr>
+  <td><code>spark.scheduler.blacklist.enabled</code></td>
+  <td>true</td>
```

Comment:
I'm actually making it true by default, except for local mode.  

https://github.com/squito/spark/blob/351a9a7e2893a0b90c57233d5e44a52c147bb2a8/core/src/main/scala/org/apache/spark/scheduler/BlacklistTracker.scala#L195

Too big a behavior change?